activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+*   Add config.active_record.dump_schemas to fix db:structure:dump
```

Comment:
can you put \` around "config.active_record.dump_schemas" and "db:structure:dump"?